File: ECE39595Homeworks/HW4Assigned/userid/PekinDuck.cpp
#include "PekinDuck.h"
#include "LaysNotBroodyEggs.h"
#include <iostream>

PekinDuck::PekinDuck(){
    quackBehavior = new Quack();
    flyBehavior = new FlyPoorly();
    laysEggs = new LaysNotBroodyEggs();
}

void PekinDuck::display(){
    std::cout<<"Pekin Duck"<<std::endl;
}

File: ECE39595Homeworks/HW14Assigned/rkothand/PekinDuck.java
public class PekinDuck extends Duck{
    public PekinDuck(){
        quackBehavior = new MuteQuack();
        flyBehavior = new FlyPoorly();
        laysEggsBehavior =new LaysEggsNotBroody();

    }

    public void display(){
        System.out.println("I am a Pekin Duck");
    }
}
